examples/panel/weights-and-biases-llm/requirements.txt

```diff
@@ -0,0 +1,5 @@
+openai==1.12.0
+panel==1.3.8
+ploomber-cloud
+python-dotenv
```

Comment:
This comes with cli: https://github.com/ploomber/cli/blob/84c70f87c0767fad28bb44fd9536e11e08d6ae39/setup.py#L21

Reply:
resolved https://github.com/ploomber/doc/pull/113/commits/edf4e9136a64843ca582325618bab2a1c65687a2